server/game/core/Constants.ts

```diff
@@ -220,6 +220,8 @@ export enum Aspect {
     Villainy = 'villainy',
 }
 
+export type AspectFilter = Aspect | Aspect[];
```

Comment:
What's the intent here? This doesn't match the other "Filter" types in here

Reply:
I was thinking this would be a shorthand for one or more aspects. It might not be used elsewhere so it's probably better to just remove it.

Reply:
I think that'd be reasonable if we didn't already have a pattern that "Filter" types don't define arrays but rather mean "a union of an enum type with its corresponding wildcard"